ecosystem/ffmpeg_plugin/mtl_st20p_rx.c

```diff
@@ -178,7 +185,17 @@ static int mtl_st20p_read_packet(AVFormatContext* ctx, AVPacket* pkt) {
   struct st_frame* frame;
 
   dbg("%s(%d), start\n", __func__, s->idx);
-  frame = st20p_rx_get_frame(s->rx_handle);
+
+  if (0 == s->frame_counter) {
```

Comment:
Please not do such dirty change. If you need increase the time out time, just customize the `timeout_s` option.
```
    {"timeout_s",
     "Frame get timeout in seconds",
     OFFSET(timeout_sec),
     AV_OPT_TYPE_INT,
     {.i64 = 0},
     0,
     60 * 10,
     DEC},
```

Reply:
Although the timeout_s will have similar functionality it will work for all packets 

This will only come in effect during initialization. 

For the purposes of initialization and the SDB case we are trying to limit the frame drops and this makes the case where the RX is dropping initial packets much less prevalent. 


Do you have maybe another suggestion for limiting the framedrop in this case?

Reply:
Do you see similar problem if using multicast address, and make sure TX is ready(stable output) before launching the RX session? Unicast is not used by any real video production setup, I could strongly suggest using multicast all the time.